### ISSUE TITLE
Add option to scroll to the top

### ISSUE THREAD

**Issue by gautamjajoo** [Cloud-CV/EvalAI-ngx#253]
## I'm submitting a
- [ ] Bug report
- [x] Feature request

## Expected behavior
Add an option to scroll to the top of the page like in the [EvalAI-cli docs](https://evalai-cli.cloudcv.org/) page.

**Comment by Suryansh5545** [comment]
Hey @gautamjajoo Can i work on it?

**Comment by drepram** [comment]
I've found the solution for it, yet I need to make a separate PR for it. Once #252 is merged I will make a new PR for this.

**Comment by Suryansh5545** [comment]
are you working on it too?

**Comment by Suryansh5545** [comment]
ok you work on it bro
